### ISSUE TITLE
[Egg Request]: <Software/Game>

### ISSUE THREAD

**Issue by Max62800** [parkervcp/eggs#2074]
### Does this expand an already existing service

Yes

### Link to a game or software

https://reignofdwarf.com

### Links for the download

https://store.steampowered.com/app/1442910/Reign_Of_Dwarf/

### Links for the install docs

For the moment none, it's coming soon I will update

**Comment by Max62800** [comment]
![image](https://user-images.githubusercontent.com/4140252/213641342-aca129ce-24b2-4f26-86af-c4db0e200410.png)

**Comment by gOOvER** [comment]
The game is not released. I i would wait, until the game have a release ;)

**Comment by Max62800** [comment]
> 

okay Thanks very much !

**Comment by gOOvER** [comment]
> > 
> 
> okay Thanks very much !

from where you have the Dev version? mb youcan ask for a dev access

**Comment by Max62800** [comment]
> > > 
> > 
> > 
> > okay Thanks very much !
> 
> from where you have the Dev version? mb you can ask for a dev access. then i can create an egg

Yes you can access the game already, I have access to the game myself
https://discord.gg/anklebreaker-studio-670344940378652673
